[RNMobile] Merge 1.46.1 beta fix release to 1.47.0

fluiddot opened the pull request:
<!-- Learn the overall process and best practices for pull requests at https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/repository-management.md#pull-requests. -->

## Description
<!-- Please describe what you have changed or added -->
This PR includes the changes that were included in the `1.46.1` beta fix to the release `1.47.0`.

## How has this been tested?
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->
N/A

## Screenshots <!-- if applicable -->
N/A

## Types of changes
<!-- What types of changes does your code introduce?  -->
<!-- Bug fix (non-breaking change which fixes an issue) -->
<!-- New feature (non-breaking change which adds functionality) -->
<!-- Breaking change (fix or feature that would cause existing functionality to not work as expected) -->
N/A

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/ -->
- [x] My code follows the accessibility standards. <!-- Guidelines: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/accessibility/ -->
- [x] I've tested my changes with keyboard and screen readers. <!-- Instructions: https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/accessibility-testing.md -->
- [x] My code has proper inline documentation. <!-- Guidelines: https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/ -->
- [x] I've included developer documentation if appropriate. <!-- Handbook: https://developer.wordpress.org/block-editor/ -->
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. <!-- React Native mobile Gutenberg guidelines: https://github.com/WordPress/gutenberg/blob/HEAD/docs/contributors/native-mobile.md -->

fluiddot committed:
Release script: Update react-native-editor version to 1.46.1

fluiddot committed:
Release script: Update with changes from 'npm run core preios'

fluiddot committed:
Update react-native-editor CHANGELOG

ceyhun committed:
[RNMobile] Add capability to bridge to show/hide audio block (#28952)

Commit:
[RNMobile] Ignore column width attribute when empty (#29015)

fluiddot committed:
Merge branch 'rnmobile/release_1.47.0' into rnmobile/merge_release_1.46.1_to_1.47.0

# Conflicts:
#	packages/react-native-aztec/package.json
#	packages/react-native-bridge/package.json
#	packages/react-native-editor/CHANGELOG.md
#	packages/react-native-editor/ios/Podfile.lock
#	packages/react-native-editor/package.json

fluiddot committed:
Update react-native-editor CHANGELOG

antonis committed:
Fixes minor changelog alignement issue

antonis reviewed (approved):
Great job @fluiddot 👍 
Thank you for tackling this. I did a quick sanity and did not notice any (new) issue.
The code changes LGTM 🎉

dcalhoun reviewed (approved):
LGTM. I cloned the branch and poked around a bit in the demo app. I did not encounter any unexpected issues.

dcalhoun committed:
Add Stories bridge methods for iOS (#29083)

Add missing bridge methods for the Stories block. The absence of these
threw an error, even though Stories block isn't fully supported for iOS
just yet.
